### DISCUSSION TITLE
No reply on appeal

### DISCUSSION THREAD

**Discussion by woto** [community/community#64262] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi, I sent the appeal form about a week ago. And didn't get any response. What should I do next? How long should I wait? And should I?

https://docs.github.com/en/site-policy/other-site-policies/github-and-trade-controls#how-is-github-ensuring-that-folks-not-living-in-andor-having-professional-links-to-the-sanctioned-countries-and-territories-still-have-access-or-ability-to-appeal

**Comment by Akash1134** [comment, marked as answer]
Hi @woto The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
